APIManager.js

```diff
@@ -0,0 +1,72 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {
+            friendsList:[],
```

Comment:
you do not need to initialize these keys here. You can just add them in the methods below when you need them. It will make your code look a lot more clean

you just need `this.data = {}`